app/models/branch_of_service_contact.rb

```diff
@@ -0,0 +1,14 @@
+class BranchOfServiceContact < ApplicationRecord
+  validates :branch, presence: true
+
+  def retiree_contact_info?
+    retiree_dsn || 
+    retiree_tel_comm || 
+    retiree_tel_tollfree || 
+    retiree_email || 
+    retiree_fax_dsn || 
+    retiree_fax_comm || 
+    retiree_fax_tollfree || 
+    retiree_post
+  end
+end
```

Comment:
So RuboCop isn't terribly happy with this new method. We could either inform RuboCop to ignore this method _or_… refactor it to something like:

```rb
class BranchOfServiceContact < ApplicationRecord
  validates :branch, presence: true

  def retiree_contact_info?
    retiree_contact_telephone? || retiree_contact_fax? || retiree_email || retiree_post
  end

  private

  def retiree_contact_fax?
    retiree_fax_dsn || retiree_fax_comm || retiree_fax_tollfree
  end

  def retiree_contact_telephone?
    retiree_dsn || retiree_tel_comm || retiree_tel_tollfree
  end
end

```